staging/src/kubevirt.io/api/flavor/v1alpha1/types.go

```diff
@@ -35,9 +35,8 @@ type VirtualMachineFlavor struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	// +listType=map
-	// +listMapKey=name
-	Profiles []VirtualMachineFlavorProfile `json:"profiles"`
+	// VirtualMachineFlavorSpec for the flavor
+	Spec *VirtualMachineFlavorSpec `json:"spec"`
```

Comment:
`Spec` should not be a pointer, because it is not an optional filed.

Reply:
ACK I'll fix this now, thanks!